docs/hardware.rst

```diff
@@ -63,12 +63,12 @@ organization for both technical and legal reasons:
 * In addition, attackers with legal authority such as law
   enforcement agencies may (depending on the jurisdiction) be able
   to compel physical access, potentially with a gag order attached,
-  meaning that the 3rd party hosting your servers or VMs may be
+  meaning that the third party hosting your servers or VMs may be
```

Comment:
third-party

Reply:
Again, "third" is an adjective here. As in, *what kind of party? A "third" one.*

The hyphenated "third-party" form is itself an adjective. As in, *What kind of service? A third-party one.*

Here, a party is hosting servers. What kind of party? A third one.